app/src/main/res/layout/list_bp_history_item.xml

```diff
@@ -3,7 +3,10 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content">
+  android:layout_height="wrap_content"
+  android:clickable="true"
+  android:focusable="true"
+  android:background="?attr/selectableItemBackground">
```

Comment:
Not necessary to fix in this PR, but this commit is actually doing two things:

- Adds a fix for the edit button being clickable even for non-editable BPs
- Adds a touch selector to the item

Since we need to keep commits small and focused (one commit, one change), this should ideally be another commit.

Reply:
Split this commit into 2